app/controllers/assessor_interface/assessment_sections_controller.rb

```diff
@@ -33,6 +29,11 @@ def load_assessment_section
             }
           )
           .find_by!(key: params[:key])
+
+      @assessment = @assessment_section.assessment
```

Comment:
I don't think we need all these for the update do we? I've actually changed this a bit in the stuff I'm doing so not a biggy if you want to merge it as is.

Reply:
We do unfortunately, to be able to render the `:show` template if the validation fails:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/b6352222fbe271f98d8b0996947d799e69e8d884/app/controllers/assessor_interface/assessment_sections_controller.rb#L19

I did wonder about refactoring this controller anyway with a view object, and it sounds like maybe your stuff will do that, so I'll merge it in for now.